browser/src/UI/components/QuickInfoContainer.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react"
+
+import { QuickInfoContainer, QuickInfoDocumentation, QuickInfoTitle } from "./QuickInfo"
+
+import TokenThemeProvider from "./../../Services/SyntaxHighlighting/TokenThemeProvider"
+
+interface IQuickInfoProps {
+    titleAndContents: ITitleAndContents
+}
+
+interface ITitleAndContents {
+    title: {
+        __html: string
+    }
+    description: {
+        __html: string
+    }
+}
+
+class QuickInfoHoverContainer extends React.Component<IQuickInfoProps> {
+    public render() {
+        const { titleAndContents } = this.props
+        const hasTitle = !!(titleAndContents && titleAndContents.title.__html)
+        const hasDocs =
+            hasTitle &&
+            !!(
+                titleAndContents &&
+                titleAndContents.description &&
+                titleAndContents.description.__html
+            )
+
+        return (
+            <TokenThemeProvider
```

Comment:
Cool idea to separate the logic of getting the token colors from the logic of rendering